bpf/lib/nodeport.h

```diff
@@ -1944,6 +1953,13 @@ static __always_inline int rev_nodeport_lb4(struct __ctx_buff *ctx, int *ifindex
 	ret = ct_lookup4(get_ct_map4(&tuple), &tuple, ctx, l4_off, CT_INGRESS, &ct_state,
 			 &monitor);
 
+	/* If reverse NAT happens on egress of the native device,
+	 * rev_nodeport_lb4 is called for all outboud traffic.
+	 * hence, check this is actually NodePort traffic.
+	 */
```

Comment:
```suggestion
	/* When we do the rev-DNAT for a NodePort reply from a local service endpoint on the bpf_host's "to-netdev" instead of bpf_lxc, then this function is executed all packets. Return early if we detect that a packet is not the reply to avoid unnecessary waste of resources. */ 
```

Reply:
Updated comment.